### ISSUE TITLE
Integration with upsource

### ISSUE THREAD

**Issue by Vaelor** [danger/danger-js#327]
Upsource does not have a pull request system on it's own, it is more of a code review tool, but I guess that should be no problem.

I would like to implement this, but I have a bit of a hard time to figure out where to get started.
Are the methods found in https://github.com/danger/danger-js/blob/master/source/platforms/FakePlatform.ts what a possible upsource connector should implement?

**Comment by orta** [comment]
Yes, close want to implement [the `Platform` interface](https://github.com/danger/danger-js/blob/master/source/platforms/platform.ts#L31),  and [edit this function](https://github.com/danger/danger-js/blob/master/source/platforms/platform.ts#L56) to return your upsource object when the time is right
